pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -657,132 +658,190 @@ func (r *ClickHouseReader) GetServicesList(ctx context.Context) (*[]string, erro
 	return &services, nil
 }
 
-func (r *ClickHouseReader) GetServices(ctx context.Context, queryParams *model.GetServicesParams) (*[]model.ServiceItem, *model.ApiError) {
-
-	if r.indexTable == "" {
-		return nil, &model.ApiError{Typ: model.ErrorExec, Err: ErrNoIndexTable}
-	}
+func (r *ClickHouseReader) GetTopLevelOperations(ctx context.Context) (*map[string][]string, *model.ApiError) {
 
-	serviceItems := []model.ServiceItem{}
+	operations := map[string][]string{}
+	query := fmt.Sprintf(`SELECT DISTINCT name, serviceName FROM %s.%s`, r.traceDB, signozTopLevelOperations)
```

Comment:
@srikanthccv `signozTopLevelOperations` could have be added to `ClickHouseReader` struct.

Reply:
There is a mixed usage of table names from constants and reader struct. I agree it's good to refer them just from reader. Will update.

Reply:
done